### DISCUSSION TITLE
Mektup oyunu sorusu

### DISCUSSION THREAD

**Discussion by sonsuzus** [sonsuzus/sonsuzus#28] [Zeka Sorusu]
Bazıları oyun oynamayı seven, bazıları ise oyun oynamayı sevmeyen 150 kişilik bir toplulukta zincirleme mektup gönderme oyunu oynanıyor. İlk başta Ali dört kişiye mektup gönderiyor. Mektubu alan herkesten mektubu Ali dışındaki dört kişiye göndermeleri isteniyor. Topluluktaki bazı kişiler oyuna ilgi gösteriyor ve mektubu aldıklarında istendiği gibi Ali dışındaki dört kişiye gönderiyor. Oyuna ilgi göstermeyenler ise mektubu hiç kimseye göndermiyor. Yeterince uzun süre beklendikten sonra hiç kimsenin birden fazla mektup almadığı ve tam 100 kişinin ise mektubu almalarına rağmen kimseye göndermedikleri anlaşılıyor. Öyleyse eline mektup geçmemiş kaç kişi vardır?

**Comment by yigidi** [comment]
Sorunuzu adım adım çözebiliriz:

1. Başlangıçta Ali dört kişiye mektup gönderiyor. Bu dört kişi, mektubu alanlar arasında sayılmaz, çünkü zaten mektup almadan önce varolan kişilerdir.

2. Geriye 150 - 4 = 146 kişi kalıyor.

3. Mektubu alan her bir kişi, Ali dışındaki dört kişiye mektup gönderiyor. Bu durumda her mektup alan kişi 4 kişiye mektup gönderiyor.

4. Topluluktaki bazı kişiler mektubu aldıklarında istendiği gibi mektubu gönderiyor, bazıları ise göndermiyor.

5. Sonunda hiç kimsenin birden fazla mektup almadığı belirtiliyor. Bu durumda her kişi yalnızca bir mektup almış olabilir.

6. Tam 100 kişi mektubu almasına rağmen kimseye göndermemiş. Bu durumda 100 kişi mektup aldı, ancak bu kişilerden hiçbiri mektubu göndermedi.

7. Geriye kalan kişi sayısı, 146 (ilk mektup gönderiminden kalanlar) - 100 (mektup alan fakat göndermeyenler) = 46 kişi.

Sonuç olarak, eline mektup geçmemiş olan kişi sayısı 46'dır.
